Criteria database

bastelfreak opened the pull request.

Commit:
Edit requirements

Edit the Requirments in the Documentation. Hope this is correct. If not,
please let me know.

Commit:
Modify the edit of the requirements

I have change and modify the comments of @bastelfreak. Please let me
know, if you find anything which is incorrect.

Commit:
Criteria's Database

First example of the criterias for evaluating the database system

Commit:
Modify the criteria for Database

I have reviewed and modify the sheet. Please let me know.

StoneCat commented:
Please paste the delete text into a new category named "bus system" or something like that

Commit:
Create the bus_system
I have copy the deleted lines from the criteria Database sheet into the
new file.

Commit:
add subject

bastelfreak commented:
dicussed with @StoneCat
